llvm/lib/Target/AArch64/AArch64ExpandPseudoInsts.cpp

```diff
@@ -64,8 +64,8 @@ class AArch64ExpandPseudo : public MachineFunctionPass {
                 MachineBasicBlock::iterator &NextMBBI);
   bool expandMultiVecPseudo(MachineBasicBlock &MBB,
                             MachineBasicBlock::iterator MBBI,
-                            TargetRegisterClass ContiguousClass,
-                            TargetRegisterClass StridedClass,
+                            const TargetRegisterClass &ContiguousClass,
+                            const TargetRegisterClass &StridedClass,
```

Comment:
TargetRegisterClass certainly should never be passed by value. The actual pointer value is significant